.pre-commit-hooks.yaml

```diff
@@ -6,6 +6,9 @@
 -   id: ansible-lint
     name: Ansible-lint
     description: This hook runs ansible-lint.
-    entry: ansible-lint
+    entry: ansible-lint --force-color .
     language: python
-    files: \.(yaml|yml)$
+    # do not pass files to ansible-lint, see:
+    # https://github.com/ansible/ansible-lint/issues/611
+    pass_filenames: false
+    always_run: true
```

Comment:
this ~usually isn't the setting you want here -- this'll do a lot of unnecessary work on every commit (linting your entire codebase all the time) -- generally you want to optimize for quickness on commit and being thorough in CI (via `--all-files`, etc.)